src/main/java/duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDate at;
```

Comment:
Similar to `Deadline.java`, you could consider a more descriptive variable name like `atDate`